src/redux/studio-permissions.js

```diff
@@ -54,6 +54,17 @@ const selectCanRemoveManager = (state, managerId) =>
     !selectIsMuted(state) && (selectIsAdmin(state) || isManager(state)) && managerId !== state.studio.owner;
 const selectCanPromoteCurators = state => !selectIsMuted(state) && isManager(state);
 
+const selectCanTransfer = (state, managerId) => {
+    if (state.studio.classroomId !== null) return false; // Nobody can transfer class studios
```

Comment:
Let's clarify this comment further, since classroomId is only set in some cases.